Src/driver.c

```diff
@@ -307,6 +360,55 @@ static void stepperCyclesPerTick (uint32_t cycles_per_tick)
 
 // Set stepper pulse output pins
 // NOTE: step_outbits are: bit0 -> X, bit1 -> Y, bit2 -> Z...
+#ifdef SQUARING_ENABLED
+inline static __attribute__((always_inline)) void stepperSetStepOutputs (axes_signals_t step_outbits_1)
+{
+	axes_signals_t step_outbits_2;
+
+
+#if STEP_OUTMODE == GPIO_BITBAND
+    step_outbits_1.mask = (step_outbits_1.mask & motors_1.mask) ^ settings.steppers.step_invert.mask;
+
+
+    BITBAND_PERI(X_STEP_PORT->ODR, X_STEP_PIN) = step_outbits_1.x;
+    BITBAND_PERI(Y_STEP_PORT->ODR, Y_STEP_PIN) = step_outbits_1.y;
+    BITBAND_PERI(Z_STEP_PORT->ODR, Z_STEP_PIN) = step_outbits_1.z;
+
+ #ifdef A_AXIS
+    BITBAND_PERI(A_STEP_PORT->ODR, A_STEP_PIN) = step_outbits_1.a;
+ #endif
+ #ifdef B_AXIS
+    BITBAND_PERI(B_STEP_PORT->ODR, B_STEP_PIN) = step_outbits_1.b;
+ #endif
+ #ifdef C_AXIS
+    BITBAND_PERI(C_STEP_PORT->ODR, C_STEP_PIN) = step_outbits_1.c;
+ #endif
+
+#elif STEP_OUTMODE == GPIO_MAP
+    STEP_PORT->ODR = (STEP_PORT->ODR & ~STEP_MASK) | step_outmap[step_outbits_1.value];
+#else
+    STEP_PORT->ODR = (STEP_PORT->ODR & ~STEP_MASK) | ((step_outbits_1.mask ^ settings.steppers.step_invert.mask) << STEP_OUTMODE);
+#endif
+
+
+#if defined(X2_STEP_PIN) || defined(Y2_STEP_PIN) || defined(Z2_STEP_PIN)
+    step_outbits_2.mask = (step_outbits_1.mask & motors_2.mask) ^ settings.steppers.step_invert.mask;
```

Comment:
There is no need for this #if test as it is always true (SQUARING_ENABLED)

Reply:
Removed